RELEASE.md

```diff
@@ -1,5 +1,31 @@
 # Release Process
 
+
+## What do we mean when we say own a release 
+
+Being assigned as the release steward means you expected to deliver 
+
+* 1 or more sets of beta releases.
+* The final release of packages. 
+* Release notes on github for every package (automatically created by changesets bot)
+* If the release is significant a forum post with the highlights and link back to release notes
+
+### What is a signicant release
+
+This is intentional not defined. If it has been a long time, if someone will be asking about the release, if there are big fixes. 
+
+
+### What do I do? 
+
+As the release steward you should be making sure all prs get the approval they need and have attention that the ci checks are not stuck. This may be automated but its not a passive role. When changesets has published try out installing the new version yourself for each beta and the final. Give celocli a quick spin. Are there people who reported bugs? Message them or comment on their github issue asking them to try the latest beta. 
+
```

Comment:
If there are any bugs this release should fix send them a message when the beta is out asking them to see if ti fixes their issue.